src/taco-remote/lib/buildManager.ts

```diff
@@ -431,6 +432,17 @@ class BuildManager {
                 } else {
                     self.buildMetrics.failed++;
                 }
+                
+                utils.TelemetryHelper.generate("build",
```

Comment:
My understanding is that `telemetry.generate` is intended for timing things. Should we start the telemetry.generate outside the `buildInfo["pkg"].build` and have it time the build?